src/platform/tls_schannel.c

```diff
@@ -1003,7 +1087,12 @@ CxPlatTlsSspiNotifyCallback(
         CompletionCallback(&CredConfig, CompletionContext, SecStatusToQuicStatus(Status), NULL);
         CxPlatTlsSecConfigDelete(SecConfig); // *MUST* be last call to prevent crash in platform cleanup.
     } else {
-        CompletionCallback(&CredConfig, CompletionContext, QUIC_STATUS_SUCCESS, SecConfig);
+        Status = (SECURITY_STATUS)QUIC_STATUS_SUCCESS;
+        if (SecConfig->Flags & QUIC_CREDENTIAL_FLAG_DEFER_CERTIFICATE_VALIDATION &&
+            SecConfig->Flags & QUIC_CREDENTIAL_FLAG_REQUIRE_CLIENT_AUTHENTICATION) {
+            Status = (SECURITY_STATUS)CxPlatTlsSetClientCertPolicy(SecConfig);
+        }
```

Comment:
Didn't we decide we were going to do this regardless of the `DEFER_CERTIFICATE_VALIDATION` flag being present?